app/views/declaration/summary/sections/countries_section.scala.html

```diff
@@ -45,8 +45,8 @@
                 ),
                 value = Value(
                     content = Text(
-                        if(declarationData.locations.routingCountries.isEmpty){
-                            messages("declaration.summary.countries.routingCountries.none")
```

Comment:
Can these deprecated message keys be removed from the messages file?

Reply:
Did not I already remove them?